z3c/dependencychecker/dependencychecker.py

```diff
@@ -501,9 +514,9 @@ def imports_from_doctests(path):
     for path, dirs, files in os.walk(path):
         for filename in [os.path.abspath(os.path.join(path, filename))
                          for filename in files
-                         if fnmatch.fnmatch(filename, '*.txt')
-                         or fnmatch.fnmatch(filename, '*.rst')
-                         or fnmatch.fnmatch(filename, '*.py')]:
+                         if fnmatch.fnmatch(filename, '*.txt') or
+                         fnmatch.fnmatch(filename, '*.rst') or
```

Comment:
Is this pep8-suggested? I like the original a bit more, visually, but I know that "nice visual effect" isn't always the best reason :-)